config/hardware/probes/tap_probe.cfg

```diff
@@ -1,6 +1,6 @@
 [gcode_macro _USER_VARIABLES]
 variable_probe_type_enabled: "vorontap"
-variable_startprint_actions: "bed_soak", "chamber_soak", "clean", "tilt_calib", "z_offset", "bedmesh", "extruder_heating", "purge", "clean", "primeline"
+variable_startprint_actions: "bed_soak", "chamber_soak", "clean", "tap_heating", "tilt_calib", "z_offset", "bedmesh", "extruder_heating", "purge", "clean", "primeline"
```

Comment:
Can you change the name of the module to "extruder_preheating" to add genericity as it could be useful for other kind of probes also.

Reply:
Renamed as suggested